src/main/java/wooteco/subway/domain/fare/ageStrategy/Age.java

```diff
@@ -0,0 +1,28 @@
+package wooteco.subway.domain.fare.ageStrategy;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+public enum Age {
+    PREFERENTIAL (age -> age < 6 || 65 <= age, new PreferentialDiscountPolicy()),
+    CHILDREN (age -> 6 <= age && age < 13, new ChildrenDiscountPolicy()),
+    TEENAGER (age -> 13 <= age && age <= 18, new TeenagerDiscountPolicy()),
+    NORMAL (age -> 19 < age, new NormalAgeDiscountPolicy())
```

Comment:
저는 보통 변수를 왼쪽에 놓고 사용하기는 합니다...!

왼쪽에서 선언된 변수를 보고  age >= 13을 읽고 age <= 18에서 머리속에서 자연스레 age를 생략하고 <= 18를 읽는 방식으로 읽는게 자연스럽더라구요 😄 
아직 정확하게 설명할 수는 없지만, 애쉬가 원하는 방식으로 작성하시면 될 것 같아요 !

```suggestion
    PREFERENTIAL (age -> age < 6 || age >= 65, new PreferentialDiscountPolicy()),
    CHILDREN (age -> age >= 6 && age < 13, new ChildrenDiscountPolicy()),
    TEENAGER (age -> age >= 13 && age <= 18, new TeenagerDiscountPolicy()),
    NORMAL (age -> age > 19, new NormalAgeDiscountPolicy())
```